### PR TITLE
chore: change targetRevision for devsecops kube-prometheus-stack appset

### PR DESCRIPTION
**Pull request by tomaszbarwicki**
PR to prepare and test applicationset at devsecops for kube-prometheus-stack upgrade.

### CODE REVIEW

File: environments/core/applicationsets/kube-prometheus-stack-applicationset.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ spec:
         targetRevision: HEAD
       - cluster: devsecops-testing
         cluster-name: devsecops-testing
-        targetRevision: HEAD
+        targetRevision: chore/devsecops_kube_upgrade
```
Comment (FaGru3n) [inline environments/core/applicationsets/kube-prometheus-stack-applicationset.yaml:20]:
```suggestion
        targetRevision: chore/devsecops_kube_prometheus_stack_appset_upgrade
````

should this one not changed to `chore/devsecops_kube_prometheus_stack_appset_upgrade` ?

Comment (tomaszbarwicki) [reply environments/core/applicationsets/kube-prometheus-stack-applicationset.yaml:20]: chore/devsecops_kube_prometheus_stack_appset_upgrade branch is for ApplicationSet change but chore/devsecops_kube_upgrade is for actual version change of the kube prometheus stack (Chart.yaml will be modified there)

Comment (FaGru3n) [reply environments/core/applicationsets/kube-prometheus-stack-applicationset.yaml:20]: kay no problem, but `chore/devsecops_kube_upgrade` seems not available in the repo so was confused. so i guess need a little push 😉 that argo will be have a target revision to checkout for devsecops cluster

**Review by FaGru3n — APPROVED** [review]
LGTM

**Pull request merged**